tracing/opentracing/options.go

```diff
@@ -23,11 +24,15 @@ type FilterFunc func(ctx context.Context, fullMethodName string) bool
 // UnaryRequestHandlerFunc is a custom request handler
 type UnaryRequestHandlerFunc func(span opentracing.Span, req interface{})
 
+//OpNameFunc is a func that allows custom operation names instead of the gRPC method.
```

Comment:
Please add a space after the slashes.
```suggestion
// OpNameFunc is a func that allows custom operation names instead of the gRPC method.
```